global/scripts/CorrectVolumeOrientation

```diff
@@ -0,0 +1,232 @@
+#   and/or other materials provided with the distribution.
+# * Neither the name of this project nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+set -eu
+
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+    pipedirguessed=1
+    #fix this if the script is more than one level below HCPPIPEDIR
+    export HCPPIPEDIR="$(dirname -- "$0")/../../"
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"
+
+# Usage
+opts_SetScriptDescription "CORRECT VOLUME ORIENTATION
+This script corrects or changes orientation of the NIFTI volume. It is useful to correct the orientation information of the NIFTI header, for example, 1) when the data was collected with the 'Patient position' incorrectly entered in the scanner or 2) when the true patient position can not be entered by the scanner (e.g. the patient or animal is in sphinx position, while the scanner cannot accept it). True patient positions available are 'head-first-sphinx (HFSx)', 'foot-first-sphinx (FFSx)', 'head-first-supine (HFS)' and 'foot-first-supine (FFS)'.
+
+Example protocols and usages:
+
+Ex) #1. the simplest usage is when an MRI volume (MRI.nii.gz) is scanned in the true position of HFSx but the scanner's Patient Position is set to HFP.
+
+$ CorrectVolumeOrientation --in=MRI.nii.gz --tposition=HFSx --sposition=HFP --out=MRI2reorient.nii.gz
+
+Ex) #2. structural MRI scanned in true position of HFS on day1 (T1w_d1_HFS) and fMRI scanned in true posiiton in HFSx on day2 (fMRI_d2_HFSx) and additional fast T1w scanned for registration purpose on day2 (T1w_d2_HFSx), where scanner's patient orientaion is set to HFS on day1 and HFP on day2. In this case, the fast T1w scanned on day2 is corrected for the head orientation and registrated to the T1w on day1, then the fMRI data on day2 is corrected for head orientation and initialized for head position using fast T1w scanned on day2. In this case, the option --ref is useful to 'fake' the output volume as if it were scanned on the same position as in day1. 
```

Comment:
It would probably be a good idea to add manual word wrapping to this entire description, as `newops.shlib` doesn't apply word wrapping code to it (only the argument descriptions get word wrapped, and that code is specialized to indent wrapping to avoid confusion with the next argument).